client/src/shared-modules/Footer.js

```diff
@@ -1,69 +1,85 @@
-/* eslint-disable no-trailing-spaces */
-/* eslint-disable linebreak-style */
 
 import React from "react";
 import { useState, useEffect } from "react";
 
 import "../pages/Footer.css";
 
 function Footer() {
-
-  const [pages, setPages] = useState([]);
+	const [pages, setPages] = useState([]);
 
 	async function fectPages() {
 		try {
-			const response = await fetch("/api/pages"
-			);
+			const response = await fetch("/api/pages");
 			const pagesFromDatabase = await response.json();
 			setPages(pagesFromDatabase);
 		} catch (err) {
 			console.error(err);
 		}
 	}
 
+
+	
 	useEffect(() => {
 		fectPages();
 	}, []);
 
+	console.log(pages.map((p) => p.page_title));
 
-
-  return (
+	return (
 		<div className="card-footer">
 			<section className="footer-address-section">
 				<img src="#" alt="The Collective Foundation logo" />
```

Comment:
Here comes the dead image of icon. You should pass the src path instead of # here